dev-libs/libugpio/libugpio-0.0.6.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit linux-info
+
+DESCRIPTION="Lib for the use of linux kernel's sysfs gpio interface from C programs"
+HOMEPAGE="https://github.com/mhei/libugpio"
+SRC_URI="https://github.com/mhei/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="static"
```

Comment:
Same question.

Reply:
It builds a static lib, but lcdproc itself doesn't need a static one. Just added it, since the build system supports it? Commited now a new one with disabled static.